build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -520,6 +520,49 @@ message Platform {
   // The server MAY use the `value` of one or more properties to determine how
   // it sets up the execution environment, such as by making specific system
   // files available to the worker.
+  //
+  // The following standard `name`s are defined:
+  //
+  // * `OS`: This describes the operating system. Only a single value is allowed
+  //   and an exact match is required. The following standard values are
+  //   defined:
+  //
+  //     - AIX
+  //     - FreeBSD
+  //     - Linux
+  //     - MacOS
+  //     - SunOS
+  //     - Windows
+  //
+  // * `ISA`: This describes the instruction set architecture including
+  //   instruction set extensions. Multiple values are allowed. The worker is
+  //   required to support all of the listed values. The following standard
```

Comment:
> "The worker is required to support all of the listed values." 

Maybe phrase as "If multiple values are specified, they are AND-ed together: the worker is required to support all of the listed values".

Otherwise, it sounds like the comment is saying "if a value is specified, the server is required to ensure the worker supports it", rather than specifying the logic for combining multiple, which contradicts the higher level comments that it's up to the server how to handle keys they don't understand (one allowed option being to ignore them entirely).

Reply:
Sounds good, will include this.